### ISSUE TITLE
RedirectTrailingSlash undefined (type *atreugo.Atreugo has no field or method RedirectTrailingSlash)

### ISSUE THREAD

**Issue by milchinskiy** [savsgio/atreugo#47]
RedirectTrailingSlash undefined (type *atreugo.Atreugo has no field or method RedirectTrailingSlash)

```
import "github.com/savsgio/atreugo/v8"

server := atreugo.New(atreugoConfig)
server. RedirectTrailingSlash(false)
```

```
$ go build
$ server.RedirectTrailingSlash undefined (type *atreugo.Atreugo has no field or method RedirectTrailingSlash)
```

atreugo version is:
	
> github.com/savsgio/atreugo v5.4.0+incompatible
> github.com/savsgio/atreugo/v8 v8.2.1

**Comment by savsgio** [comment]
Which version are you using?

Remove `github.com/savsgio/atreugo v5.4.0+incompatible` from your `go.mod`

**Comment by milchinskiy** [comment]
v8.2.1
when I delete it, the next time I 'go get' it's added again

**Comment by savsgio** [comment]
Make sure that you have the correct import `import "github.com/savsgio/atreugo/v8"` in all go files

**Comment by milchinskiy** [comment]
Yeah, I'm sure.

**Comment by savsgio** [comment]
Remove `go.sum` and exec `go get`

Could you share your code?

**Comment by milchinskiy** [comment]
sorry, I cannot share my code. Removed go.sum, cleared up go.mod, `go get` and the same thing... I use go 1.13 if it is make sense.

**Comment by savsgio** [comment]
Ooohh! forgive me. I've remembered that the version 8.2.1 doesn't have `server.RedirectTrailingSlash(false)`, now it's just in master brach.

But don't worry.
I will release the version `9.0.0 soon!

**Comment by milchinskiy** [comment]
ok, I'll wait 9.0.0

**Comment by savsgio** [comment]
I've just released [v9.0.0](https://github.com/savsgio/atreugo/releases/tag/v9.0.0)
